ports/rubberband/portfile.cmake

```
@@ -0,0 +1,49 @@
+vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    FEATURES
+      cli CLI
+      lv2 LV2
+)
+
+vcpkg_download_distfile(
+  ARCHIVE URLS "https://breakfastquay.com/files/releases/rubberband-2.0.2.tar.bz2"
+  FILENAME "rubberband-2.0.2.tar.bz2"
+  SHA512 87dc5d59995bf949c35b885bb2eced067d84775bab76692da1202ae7554cb1b8404ca7371c264363e6d55c8978762ed32b446535746061f09e90cfc599adc0a0
+)
+
+vcpkg_extract_source_archive_ex(
+  OUT_SOURCE_PATH SOURCE_PATH
+  ARCHIVE ${ARCHIVE}
+)
+
+vcpkg_configure_meson(
+    SOURCE_PATH ${SOURCE_PATH}
```

```suggestion
    SOURCE_PATH "${SOURCE_PATH}"
```
Please also add double quotes to the absolute paths.